cosi-driver/pkg/server/driverserver.go

```diff
@@ -0,0 +1,47 @@
+package server
+
+import (
+	"fmt"
+
+	"github.com/container-object-storage-interface/spec/lib/go/cosi"
+	"github.com/golang/glog"
+	"golang.org/x/net/context"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+)
+
+type DriverServer struct {
+	Name, Version string
+}
+
+func (ds DriverServer) ProvisionerGetInfo(context.Context, *cosi.ProvisionerGetInfoRequest) (*cosi.ProvisionerGetInfoResponse, error) {
```

Comment:
These should be (ds *DriverServer) I believe.  Is there a reason for a value receiver vs over a pointer receiver?

Reply:
Yep, moved to pointer receiver. No need to pass value object.